Util/ViolationFormatter.php

```diff
@@ -11,40 +11,11 @@
 
 namespace FOS\RestBundle\Util;
 
-use FOS\RestBundle\Controller\Annotations\Param;
-use FOS\RestBundle\Controller\Annotations\QueryParam;
-use Symfony\Component\Validator\ConstraintViolationInterface;
-use Symfony\Component\Validator\ConstraintViolationListInterface;
+use FOS\RestBundle\Validator\ViolationFormatter as BaseViolationFormatter;
 
-class ViolationFormatter implements ViolationFormatterInterface
+/**
+ * @deprecated since 1.7, to be removed in 2.0. Use {@link BaseViolationFormatter} instead.
```

Comment:
using the FQCN would make the message more understandable when being logged by the DebugClassLoader